src/nvim/msgpack_rpc/server.c

```diff
@@ -224,7 +224,7 @@ int server_start(const char *endpoint)
 /// @param endpoint Address of the server.
 void server_stop(char *endpoint)
 {
-  Server *server;
+  Server *server = NULL;
```

Comment:
In fact, refactoring ga_len to `size_t` is something we should do at some point. Apart from causing subtle bugs like this, it's also the main cause for a lot of casting between int and size_t. It's something I have in my type-refactoring-todo list, but I haven't reached it yet.